### ISSUE TITLE
508-Defect-3 [HTML] Find A VA Form search results fails Accessibility Test Case 1.3.1 Info and Relationships

### ISSUE THREAD

**Issue by noahgelman** [department-of-veterans-affairs/va.gov-team#28258]
[508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## This Ticket is for This Project
https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/27483

## Failing Test Case
1.3.1 Info and Relationships
https://dsvavsp.testrail.io/index.php?/tests/view/205466&group_by=cases:section_id&group_order=asc&group_id=1367

## Point of Contact

**VFS Point of Contact:** Noah

## Details

Search results are not announcing the number of results correctly.

A search result for "EZ" has 4 results but screen readers are announcing "A list with 19"

Each result has multiple lines of content. Those results are not organized into blocks of content. All the results and their content are on the same hierarchy which makes the screen reader think there are 19 results instead of 4.

![f1](https://user-images.githubusercontent.com/984252/128160087-f0ad6e14-161a-42a3-a17f-fd03de2b3cdf.PNG)

![f2](https://user-images.githubusercontent.com/984252/128160224-ccb5a850-44fa-4745-ad3c-d31853ce5290.PNG)

## Proposed Solution (if known)

The HTML needs to be updated to move each result into their own block of content.

## WCAG or Vendor Guidance (optional)

https://www.w3.org/TR/WCAG21/#info-and-relationships

**Comment by MarciMcGuireGCIO** [comment]
@noahgelman - just to confirm - is this a result of the new WCAG test cases? Thx!

**Comment by zacharymorel** [comment]
Here's a ticket in our current sprint where I can talked this issue as well since the cover similar issues. 
https://github.com/department-of-veterans-affairs/va.gov-team/issues/26353

**Comment by zacharymorel** [comment]
Completeing 26353 in our current sprint should solve this issue.

**Comment by noahgelman** [comment]
@zacharymorel Fantastic. Less work for you guys then 👍

**Comment by zacharymorel** [comment]
@noahgelman @pjhill 

A PR went in recently and this is ready to give it a review.

**Comment by noahgelman** [comment]
@zacharymorel @pjhill 

I am approving this in staging. Very good job 👍

**Comment by MarciMcGuireGCIO** [comment]
Thank you, @zacharymorel, and thanks for reviewing, @noahgelman !

**Comment by MarciMcGuireGCIO** [comment]
I don't see why not!
